tests/testthat/test-multitables.R

```diff
@@ -228,6 +230,12 @@ with_test_authentication({
         expect_identical(names(refresh(multitables(ds))), "Yet another name")
     })
 
+    test_that("Can copy the multitable to a new multitable", {
+        m <- importMultitable(name='copied_multitable', data=ds, multitable=mult)
+        expect_identical(name(m), "copied_multitable")
+        is.public(multitables(ds))[2] <- TRUE
```

Comment:
L235 tests that the copied multitable exists. L236 doesn't assert anything: it sets a multitable to be public. If you want to assert that a public multitable, when copied, remains public, then (a) shouldn't that be tested with unit tests on the server side, and (b) to assert that here, you'd do `expect_true(is.public(m))`